Jenkinsfile

```diff
@@ -31,9 +38,25 @@ fhBuildNode([labels: ['nodejs6-ubuntu']]) {
         gruntBuild {
             name = 'fh-dataman'
         }
+        s3PublishArtifacts([
+                bucket: "fh-wendy-builds/$COMPONENT/$BUILD",
+                directory: "./dist"
+        ])
+    }
+
+    stage('Platform Update') {
+        final Map updateParams = [
+                componentName: COMPONENT,
+                componentVersion: VERSION,
+                componentBuild: BUILD,
+                changeUrl: CHANGE_URL
+        ]
+        fhcapComponentUpdate(updateParams)
+        //ToDo Currently dataman isn't added to any openshift templates
+        //fhOpenshiftTemplatesComponentUpdate(updateParams)
```

Comment:
@pb82 @sedroche fh-dataman doesn't appear to have been added to any openshift templates repo, is that correct? Is it supposed to be?

Reply:
fh-dataman is only being released in the 3.19 release afaik. I am unsure of the 4.x release plans but there is an open PR for the template for this => https://github.com/feedhenry/fh-openshift-templates/pull/128

Reply:
@mikenairn @sedroche The PR for the 4.x template still needs another review from the core team which should happen soon.

Reply:
@sedroche @pb82 Thanks. The build triggered by this method "fhOpenshiftTemplatesComponentUpdate" currently fails if it can't find the variable in the templates repo that it's expecting to alter, in this case "FH_DATAMAN_IMAGE_VERSION". I'll update the job to get round this, and when you do eventually merge this PR https://github.com/feedhenry/fh-openshift-templates/pull/128, it will all work.